lib/twitter-ads/audiences/tailored_audience.rb

```diff
@@ -65,8 +65,8 @@ class << self
       # @since 0.3.0
       #
       # @return [TailoredAudience] The newly created tailored audience instance.
-      def create(account, file_path, name, list_type)
-        upload = TwitterAds::TONUpload.new(account.client, file_path)
+      def create(account, file_path, name, list_type, opts = {})
```

Comment:
We weren't passing options through here on purpose. The only valid option you could pass here is the `bucket` name, however for Tailored Audiences only the `ta_partner` bucket is valid for audience uploads and processing. If you want to upload to the TON API for another purpose my recommendation is to use the `TwitterAds::TONUpload` class directly.